i18n/MonotoneMuseum/ko_kr.yml

```
@@ -25,65 +25,538 @@ ko_kr:
+        - |-
+          【마비】
+          이형화한 육체의 변이에, 영혼이 버티지 못한다. 몸이 저리고 떨림이 멎지 않는다. 당신은 배드스테이터스 [마비]를 받는다.
+        - |-
+          【중독】
+          의식 깊은 곳에서 이형화가 진행되어 당신의 육체를 좀먹기 시작한다. 당신은 배드스테이터스 [사독]5를 받는다.
+        - |-
+          【색채 상실】
+          당신의 몸에서 색이 사라져 탁한 백색에 가까워진다. 옷이나 화장으로 숨기는 것은 가능하나 들키면 이단심문은 피할 수 없을테지. 당신의 박탈치를 +1d6한다 (이 효과로는 징조표가 굴러가지 않는다)
+        - |-
+          【이유 상실】
+          당신의 정신이 당신의 현재를 부정한다. 당신은 [경험]과 그것에 관한 기억을 그 씬 동안 잊는다. 특징에 의한 효과는 사라지지 않지만 그것을 인식하는 것은 불가능하다.
+        - |-
+          【존재 상실】
+          문득 연기처럼 당신의 모습이 사라진다. 당신의 존재는 잠깐동안 이 세계에서 소실된다. 당신의 박탈치를 +1D6한다 (이 효과로는 징조표를 굴리지 않는다)
+        - |-
+          【이탈의 대가】
+          이탈능력을 사용한 결과로 징조표를 굴리게 되었을 경우에만 적용한다 (그렇지 않을 경우 다시 굴린다). 이 표를 굴리는 원인이 된 이탈능력을 이 라운드 (전투중이 아니라면 씬 동안) 동안 더 이상 사용할 수 없다. 
+        - |-
+          【이탈의 책임】
+          세계에서 이탈된 결과, 육체가 급격화게 이형화하고 몸이 아파온다. [상승한 박탈치x10]점의 【HP】를 잃는다.
+        - |-
+          【급격한 이형화】
+          신체 일부(귀, 손가락, 눈) 등이 일그러진 짐승처럼 변한다. 하지만 그것은 당신의 힘이 되기도 한다. 【HP】를 (캐릭터레벨)D6점 회복한다. (회복할 수 없는 상황에서도 회복한다.) 당신의 박탈치를 +2한다. (이로 인한 징조표는 굴리지 않는다.)
+        - |-
+          【정신 붕괴】
+          몸이 이형화하고 세계에서 박탈되어간다. 당신의 존재는 흐릿해지다 이윽고 사라져간다. 영혼조차도. 당신은 [2D6+캐릭터레벨] 점의 【MP】를 잃는다.
+        - |-
+          【이탈의 저주】
+          이탈능력을 사용하여 이 표를 굴린 경우에만 적용한다(그 이외에는 다시 굴릴 것). 이 표를 굴린 원인이 된 이탈능력은 이 공연 동안 사용할 경우 박탈치 상승이 2배가 된다.
+        - |-
+          【박탈의 전파】
+          세계의 왜곡은 넓어지고 확장되어 누구도 도망칠 수 없다. 당신의 박탈치가 씬에 등장한 PC중에 가장 박탈치가 높은 PC와 같아진다. 만약 당신이 가장 높다면 가장 낮은 PC의 박탈치가 당신과 같아진다(여럿이 등장하고 있을 경우에는 랜덤으로 정할것).
+        - |-
+          【운명의 총애】
+          총애 혹은 비극의 대가. 당신은 이탈능력을 이 공연에 한해 추가로 하나 더 사용할 수 있게 된다(임의로 선택할 것). 선택한 이탈능력의 박탈치 상승은 +1된다.
+      DTO:
+        name: 일그러짐표(야외)
+        items:
+        - |-
+          【죽음만연】
+          나무는 마르고 기괴한 모습이 되며 꽃은 시든다. 대기는 탁한 썩은내를 뿜으며 바위는 뒤틀린다. 모든 것이 죽음의 기운에 감싸인다.
+        - |-
+          【왜곡표】
+          일그러진 이정표가 내려온다. 일그러짐표（『MM』P263）를 굴린다.
+        - |-
+          【계절변용】
+          계절이 갑자기 변화한다. 1D6을 굴려 1이면 봄, 2면 여름, 3이면 가을, 4는 겨울, 5라면 사계절이 미친듯이 변동되고 6이면 GM이 임의로 정한다. 급격한 계절의 급격한 변화가 자연과 사람들의 건강을 해친다.
+        - |-
+          【올풀림】
+          세계에 금이 가고 올풀림이 나타난다. 올풀림에 닿은 자는 허무에 삼켜져 돌아올 수 없다.
+        - |-
+          【이형화】
+          주변의 동물들이 일그러짐때문에 이형으로 변해 습격해온다. 이형 무리（『IZ』P237）×1D6체와 전투를 해도 좋다.
+        - |-
+          【동결세계】
+          기온이 비정상적으로 저하해 모든 것이 얼어붙는다. 또한 얼어붙은 세계에서 전투하는 모든 캐릭터는 방어판정에서 펌블이 날 경우 낭패를 받는다.
+        - |-
+          【기상폭주】
+          태풍, 용오름, 눈보라, 폭염 등 이상기후가 발생한다. 배는 나뭇잎처럼 흔들리고 짐승은 공포에 허둥댄다. 이 씬 내에서 진행하는 모든 이동의 이동 거리가 반감한다.
+        - |-
+          【하늘의 분노】
+          일그러짐에 의해서 있을 수 없는 이상기후가 발생한다. 모든것을 날려보내는 바람이 불고 뇌명이 운다. 이 씬 중 모든 캐릭터는 비행상태가 될 수 없다.
+        - |-
+          【미궁의 숲】
+          주변의 풍경이나 지형이 지식이나 지도와 전혀 맞지 않는다. 어딘가로 향할 때 【지각】 난이도 12의 판정에 실패한 경우 헤멘 결과 【MP】 2D6점을 잃는다.
+        - |-
+          【죽음의 세계】
+          세계는 기묘하도록 아름답다. 물에는 물고기 한 마리 없고 덤불에는 벌레 한 마리 없다. 하늘에는 새가 날지 않는다. 기분나쁜 정적이 세계를 감싼다.	
```

The trailing space (tab) should be removed.
```suggestion
          세계는 기묘하도록 아름답다. 물에는 물고기 한 마리 없고 덤불에는 벌레 한 마리 없다. 하늘에는 새가 날지 않는다. 기분나쁜 정적이 세계를 감싼다.
```